calendar-bundle/src/DependencyInjection/ContaoCalendarExtension.php

```diff
@@ -16,14 +16,15 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\Extension;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Webmozart\PathUtil\Path;
 
 class ContaoCalendarExtension extends Extension
 {
     public function load(array $mergedConfig, ContainerBuilder $container): void
     {
         $loader = new YamlFileLoader(
             $container,
-            new FileLocator(__DIR__.'/../Resources/config')
+            new FileLocator(Path::join(__DIR__, '../Resources/config'))
```

Comment:
I can click on `__DIR__.'/../Resources/config'` in my IDE and it will take me to the target. This does not work anymore with `Path::join()`, therefore I think we should not apply it to resolvable paths.

Reply:
Agreed. I left all these out for now.

As a sidenote: When writing new code, we should imo always consider canonicalizing paths that contain things like `__DIR__` or `__FILE__` (`Path::canonicalize(__DIR__ . '/../some/where')`) if they aren't immediately consumed by something that can surely handle them. 

This way things like `/../`, `a\b/c`, `a//b//c/`, ... won't ever appear inside a path &rarr; makes debugging and testing easier and prevents subtle mistakes. :slightly_smiling_face: